From 00ebe4e4298fb19d9b8fc78b16053fb0b92eea9f Mon Sep 17 00:00:00 2001
From: Mark Reynolds <mreynolds@redhat.com>
Date: Fri, 25 May 2018 09:47:31 -0400
Subject: [PATCH] Ticket 49576 - Add support of ";deletedattribute" in
 ds-replcheck

Description: Also need to check for ";deletedattribute" when processing LDIF file

https://pagure.io/389-ds-base/issue/49576

Reviewed by: tbordaz(Thanks!)

(cherry picked from commit 9e046a35a0f771e77c788cddee2cbddee6ae0571)
---
 ldap/admin/src/scripts/ds-replcheck | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ldap/admin/src/scripts/ds-replcheck b/ldap/admin/src/scripts/ds-replcheck
index b801ccaa8..661c9e0ce 100755
--- a/ldap/admin/src/scripts/ds-replcheck
+++ b/ldap/admin/src/scripts/ds-replcheck
@@ -197,7 +197,7 @@ def remove_attr_state_info(attr):
     idx = attr.find(';')
     if idx > 0:
         state_attr = attr  # preserve state info for diff report
-        if ";deleted" in attr:
+        if ";deleted" in attr or ";deletedattribute" in attr:
             # Ignore this attribute it was deleted
             return None, state_attr
         attr = attr[:idx]
-- 
2.17.0

